src/control/system/faultdomain.go

```diff
@@ -142,6 +145,20 @@ func (f *FaultDomain) MustCreateChild(childLevel string) *FaultDomain {
 	return child
 }
 
+// Uint64 transforms the Fault Domain into an identifying 64-bit integer.
+func (f *FaultDomain) Uint64() uint64 {
+	hasher := fnv.New64a()
```

Comment:
@mjmac already uses the package "github.com/mitchellh/hashstructure" in lib/control/storage.go, perhaps we want to use the same mechanism for all instances where we want to generate hashes?

Reply:
I talked to Mike about this a bit last week. The hashstructure package actually hashes a struct - which we could use, but I thought it would be simpler to just hash the string.